src/pybind/CMakeLists.txt

```diff
@@ -49,8 +52,13 @@ add_custom_command(OUTPUT ${NMODL_PYTHON_FILES_OUT}
                    DEPENDS ${NMODL_PYTHON_FILES_IN} $<TARGET_FILE:_nmodl>
                    COMMENT "-- COPYING NMODL PYTHON FILES --")
 
+# =============================================================================
+# Copy python binding components into build directory
+# =============================================================================
+file(GLOB NMODL_PYTHON_HELPER_FILES "${NMODL_PROJECT_SOURCE_DIR}/nmodl/*.py")
+file(COPY ${NMODL_PYTHON_HELPER_FILES} DESTINATION ${CMAKE_BINARY_DIR}/lib/python/nmodl/)
```

Comment:
Just curious.. Why was this needed?

Reply:
NMODL uses some of this stuff for sympy solvers IIRC.